examples/timeseries/main.go

```diff
@@ -2,13 +2,25 @@ package main
 
 import (
 	"log"
+	"os"
 	"time"
 
 	"github.com/unidoc/unichart"
 	"github.com/unidoc/unichart/dataset"
+	"github.com/unidoc/unichart/examples"
+	"github.com/unidoc/unipdf/v3/common/license"
 	"github.com/unidoc/unipdf/v3/creator"
 )
 
+func init() {
+	// Make sure to load your metered License API key prior to using the library.
+	// If you need a key, you can sign up and create a free one at https://cloud.unidoc.io
+	err := license.SetMeteredKey(os.Getenv(`UNIDOC_LICENSE_API_KEY`))
+	if err != nil {
+		panic(err)
+	}
+}
+
 func main() {
 	now := time.Now()
```

Comment:
might be better to fix the time so the graph doesn't always change?

Reply:
at least easier for comparisons to see it doesn't change

Reply:
I've replaced the time initialization in the latest updates.